types/jsreport__nodejs-client/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for @jsreport/nodejs-client 3.0
+// Project: https://github.com/jsreport/jsreport/tree/master/packages/nodejs-client
+// Definitions by: reediculous456 <https://github.com/reediculous456>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import JsReport = require('jsreport');
+import { ServerResponse } from 'http';
+
+declare namespace JsReportNodeJsClient {
+    interface ClientRenderResponse extends ServerResponse {
+        body(): Promise<Buffer>;
+    }
+
+    interface Client {
+        render(req: Partial<JsReport.Request>, options?: object): Promise<ClientRenderResponse>;
```

Comment:
```suggestion
        /** @async */
        render(req: Partial<JsReport.Request>, options?: object): Promise<ClientRenderResponse>;
```
https://jsdoc.app/tags-async.html#:~:text=The%20%40async%20tag%20indicates%20that%20a%20function%20is,tag%20is%20available%20in%20JSDoc%203.5.0%20and%20later.
`.d.ts` lacks ability to use `async` keyword